### ISSUE TITLE
Tests started failing (with recent Kelp-Module-Symbiosis?)

### ISSUE THREAD

**Issue by andk** [brtastic/perl-kelp-module-raisin#1]
Sample fail report: http://www.cpantesters.org/cpan/report/368c8ad2-3a32-11ef-a1af-adbc4ec43681

Statistical analysis on the results of my smokers tells us:

```
****************************************************************
Regression 'mod:Kelp::Module::Symbiosis'
****************************************************************
Name                   Theta          StdErr     T-stat
[0='const']           1.0000          0.0000    23406789658606000.00
[1='eq_2.10']        -1.0000          0.0000    -5205080051901554.00

R^2= 1.000, N= 182, K= 2
****************************************************************
```

which can be read as: all tests with Kelp::Module::Symbiosis 2.10 were failing so far, all other versions of Kelp::Module::Symbiosis produced PASS

**Comment by bbrtj** [comment]
Thank you for reporting. This is indeed a unforeseen breaking change due to switch to eager mounting of apps (was lazy before). I'll get it fixed :)

**Comment by bbrtj** [comment]
Fixed in Kelp::Module::Symbiosis 2.11
